cmake/stm32/f3.cmake

```diff
@@ -4,19 +4,19 @@ set(STM32_F3_TYPES
     F378xx F398xx 
 )
 set(STM32_F3_TYPE_MATCH 
-    "301.[68]" "302.[68]" "302.[BC]" "302.[ED]" "303.[68]" "303.[BC]" 
-    "303.[ED]" "318.." "328.." "334.[468]" "358.." "373.[8BC]"
+    "301.[68]" "302.[68]" "302.[BC]"  "302.[DE]" "303.[68]"  "303.[BC]" 
+    "303.[DE]"    "318.."    "328.." "334.[468]"    "358.." "373.[8BC]"
     "378.." "398.."    
 )
 set(STM32_F3_RAM_SIZES 
-     16K  16K   0K  64K  16K   0K  
-     80K  16K  16K  16K  48K   0K
-     32K  80K
+     16K  16K   0K  64K  12K   0K  
+     64K  16K  12K  12K  40K   0K
+     32K  64K
```

Comment:
![image](https://user-images.githubusercontent.com/29129731/102715152-9d541c80-42d3-11eb-86cf-07a17b58e328.png)

Reply:
Yes 80 is in fact 64+16 but the 16 seems not to be aliased. So they are in CCRAM section instead. Datasheet is misleading.
I also double checked with STM32CubeF3\Drivers\CMSIS\Device\ST\STM32F3xx\Source\Templates\iar\linker\stm32f398xx_flash.icf if they did the same